### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by armp30**
the project name has changed

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @armp30. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2023/07/18 12:39:35 Loaded 5988 libraries from DB
Changing URL of library ViraLink-MQTT-Client from https://github.com/viralinkio/ViraLink-Arduino-SDK.git to https://github.com/viralinkio/ViraLink-MQTT-Client.git
Success!
```

**Pull request merged**
